cypress/support/pages/checkout.pageObject.js

```diff
@@ -0,0 +1,123 @@
+import PageObject from '../PageObject';
+
+class CheckoutPageObject extends PageObject {
+  url = '/index.html';
+
+  get laptopsField() {
+    return cy.contains('Laptops');
+  }
+
+  laptopsFieldClick() {
+    this.laptopsField.click();
+  }
+
+  get laptopItem() {
+    return cy.get('a').contains('Sony vaio i7');
+  }
+
+  laptopItemClick() {
+    this.laptopItem.click();
+  }
+
+  get addToCartBtn() {
+    return cy.contains('a', 'Add to cart');
+  }
+
+  addToCartBtnClick() {
+    this.addToCartBtn.click();
+  }
+
+  alertMessage() {
+    return this.assertAllert('Product added');
```

Comment:
There seems to be a typo in the method name `assertAllert`. It should likely be `assertAlert` to correctly spell 'alert'. Please check the implementation of this method in the parent class or wherever it's defined.